site/docs/components/button/usage.mdx

```diff
@@ -42,6 +42,8 @@ import { Button } from "@salt-ds/core";
 
 - Always style button text in all caps for prominence. This avoids confusion with body text or general writing.
 - Never use ampersand (&) in button text to save space.
+- Always use verbs that emphasize the action the button carries out. For example, use Create instead of Ok to create a profile, or Save/Discard instead of Yes/No to save changes.
```

Comment:
```suggestion
- Always use verbs that emphasize the action the button carries out. For example, use **Create** instead of **Ok** to create a profile, or **Save**/**Discard** instead of **Yes**/**No** to save changes.
```